packages/docusaurus/src/commands/deploy.ts

```diff
@@ -103,10 +103,10 @@ This behavior can have SEO impacts and create relative link issues.
     shell.exit(0);
   }
 
-  // github.io indicates organization repos that deploy via master. All others use gh-pages.
+  // github.io indicates organization repos that deploy via default branch. All others use gh-pages.
   const deploymentBranch =
     process.env.DEPLOYMENT_BRANCH ||
-    (projectName.indexOf('.github.io') !== -1 ? 'master' : 'gh-pages');
+    (projectName.indexOf('.github.io') !== -1 ? 'main' : 'gh-pages');
```

Comment:
This is the main one that could actually break things. It's a pretty bad assumption to make at all though, so this isn't really any more wrong (technically).